docs/src/main/asciidoc/building-my-first-extension-guide.adoc

```diff
@@ -0,0 +1,272 @@
+
+Building my first Quarkus Extension
+-----------------------------------
+
+https://quarkus.io/extensions/[Quarkus extensions] enhance your application just as projects dependencies do. The role of the extensions is to do all the technical piping to make third-party libraries/technologies Quarkus-compatible. This is how you can use your battle-tested ecosystem and take advantage of Quarkus perfomance and native compilation
+
+Prerequisites
+~~~~~~~~~~~~~
+
+To complete this guide, you need:
+
+* less than 30 minutes
+
+* https://github.com/quarkusio/quarkus[Quarkus sources ]
+
+* an IDE
+
+* JDK 1.8+ installed with JAVA_HOME configured appropriately
+
+* Apache Maven 3.5.3+
+
+Basic Concepts
+~~~~~~~~~~~~~~
+
+First thing first, we will need to start some basic concepts.
+
+* JVM mode Vs Native mode
+  ** Quarkus is before all a Java framework, that means you can develop, package and run classic JAR application, that's what we call *JVM mode*.
+  ** Thanks to https://graalvm.org/[GraalVM] you can compile your Java application into machine specific code (like you do in Go or C++) and that's what we call *Native Mode*.
+
+The operation of compiling Java bytecode into a native system-specific machine code is named *Ahead of Time Compilation* (aka. AoT).
+
+* Build time Vs. Run time
```

Comment:
Same here about `vs`.

Reply:
I think `aka.` should either be `a.k.a.` or `aka`